lib/team.rb

```diff
@@ -0,0 +1,22 @@
+class Team
```

Comment:
Watch your indentation, should be 2 spaces not four and make sure it all lines up